pom.xml

```diff
@@ -17,9 +17,9 @@
   <url>https://github.com/jenkinsci/winstone</url>
 
   <properties>
-    <revision>5.12</revision>
+    <revision>5.11.1</revision>
```

Comment:
Do not do this sort of thing in `master`. Use e.g. https://gist.github.com/jglick/86a30894446ed38f918050c1180483e2